data/ka'het/builder weapons.txt

```diff
@@ -28,8 +28,8 @@ outfit "Builder Panic Pulse"
 		"firing force" 1000
 		"firing heat" 160
 		"blast radius" 25
-	description "Though the Emergency Deployer was suitable for most situations, the Builders realized that it resulted in a ship effectively being downed if it fired off its last or only reactor with no ships around to assist it. To alleviate this problem somewhat, they developed this simple EMP launcher that drains the energy out of a single Reserve Accumulator, firing it out as a directed pulse. This weapon never made it to testing aboard Ka'Het vessels, relegating it to Mar-series drones only."
-	description "	To keep the weapon small, the Builders forwent most EMP protections used in larger weapons like the Nullifier. As such, small craft using the weapon can find themselves disabled for a few seconds post-firing, and as such, the weapon forces the ship back a ways to distance the ship and its target."
+	description "Though the Emergency Deployer is powerful enough for most situations in which it sees action, the Builders realized that it resulted in a ship effectively being disabled if it fired off its last or only reactor with no ships around to assist it. To alleviate this problem somewhat, they developed this simple EMP launcher that drains the energy out of a single Reserve Accumulator, firing it out as a directed pulse. This weapon never made it to testing aboard Ka'het vessels, relegating it to Mar-series drones only."
+	description "	To keep the weapon small, the Builders forwent most EMP protections used in larger weapons like the Nullifier. As such, small crafts using this weapon can find themselves disabled for a few seconds post-firing; however, the also weapon forces the ship back a ways to distance the ship and its target."
```

Comment:
I'm being stubborn here, too; the "and as such" implies causality, which "however" lacks. The weapon was designed to induce recoil **because** of the self-EMPing effect.